docs/rules.md

```diff
@@ -375,6 +376,28 @@ composer require symfony/var-dumper
 .. _DOCtor-RST:     https://github.com/OskarStark/DOCtor-RST
 ```
 
+## `ensure_explicit_nullable_types`
+
+  > _Ensure explicit nullable types are used._
+
+#### Groups [`@Symfony`]
+
+##### Valid Examples :+1:
+
+```rst
+function foo(?string $bar = null)
+```
+
+```rst
+function foo(string|null $bar = null)
+```
+
+##### Invalid Examples :-1:
+
+```rst
+function foo(string $bar = null)
+```
+
```

Comment:
This file is fully generated, depending on rule attributes (Description, InvalidExample, ValidExample)

Locally I do `php bin/doctor-rst rules > docs/rules.md` to generate it

Reply:
Oh thanks! (and great DX btw 👏 )